lib/testing/src/lib/core/models/user.model.ts

```diff
@@ -20,9 +20,10 @@ import { StringUtil } from '../string.util';
 export class UserModel {
 
     firstName: string = StringUtil.generateRandomString();
-    lastName: string = StringUtil.generateRandomString();
+    lastName: string = StringUtil.generateRandomString() + 'LastName';
```

Comment:
why do we need always this LastName?

Reply:
so that we can search on a common lastname and assert that we are only seeing users that we expect to see in the dropdown. Basically to have a common suffix that we can search on.